easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+easyblock = 'PythonPackage'
```

Comment:
Are you sure they're needed? I have a local user reporting that it works fine as is i.e. it came up with the same results as he's found elsewhere.